apps/web/pages/api/registry/featured.ts

```diff
@@ -1,6 +1,6 @@
 import { NextApiRequest, NextApiResponse } from 'next';
 import { db } from 'apps/web/src/utils/ocsRegistry';
-import { kv } from '@vercel/kv';
+import { kv } from 'apps/web/src/utils/kv/kv';
```

Comment:
`.../kv/kv`

feels a little weird. 

you could put it in index.ts and drop the second `kv`

Reply:
ah, yeah, that works. thanks

Reply:
hmm, well, reason i nested in a subdir is that i'm going to need some additional utils for the data store migration, and was planning to put them in this same location. 

couple options come to mind, let me know if you have a preference:
1. rename current file to `kv/index`, add `kv/[migrationFiles]` in the future
2. remove current file from subdir, create new subdir for the migration